src/adapter/bundle.js

```diff
@@ -42,6 +44,13 @@ define([
                         "$injector"
                     ]
                 }
+            ],
+            policies: [
+                {
+                    category: "composition",
+                    implementation: AdapterCompositionPolicy,
+                    depends: [ "mct" ]
```

Comment:
Note this is currently `PublicAPI` not `mct`.  I think we renamed it in another branch but not sure where.